server/admin_management/tests/authorization/test_jwt_validation.py

```diff
@@ -155,4 +157,4 @@ def test_get_application_invalid_signature_failure(
 
     response = test_client_fixture_unit.get(f"{endPoint}", headers=headers(token))
 
-    assert_error_response(response, 401, ERROR_VALIDATION)
+    assert_error_response(response, 401, ERROR_CLAIMS)
```

Comment:
Hi Ian, just want to double check this change is what we expected?

Reply:
Glad you asked this one.
 
Before replacing with library and after replacing with new library I ran whole test sets several times, this is one of the tests that were failed and needed to be adjusted.
In fact, this is **the one** test that's giving me confusion as well.
If we checkout '**main**' branch, run the same "set" of tests (under admin_management), they all pass. If we run **individual** test (::test_get_application_invalid_signature_failure) it will fail. (the same running outcomes for after I replace to the new library).

**To minimize the impact,** the "**ERROR_CLAIMS**" is what's expected from test failure. But, actually **I would prefer removing this test** (I didn't do it), as the test result is not consistent and I feel the initial setup (in 'main' branch) probably was not even correct; but the JWT validation is complicated.

What I thought is, when the **"jwt_validation.py"** initially was written should not have used try-catch many exceptions, some of the validations in jwt_validation.py were(are) not necessary; it should just relies on the library's validation to throw its meaningful/detail exception. However, for library's throwing exception back to our backend FastAPI, then FastAPI probably won't be able to handle it and crush the backend. We should just had a general try-catch and return known "HTTPException" that FastAPI is able to handle and return the response.
However, to change this will change quite some codes (and tests) and will impact a lot. 
So at the end, for minimize the impact of this pr, the "**ERROR_CLAIMS**" is what expected for now.

Or do you feel you also prefer to remove this test, or change the code (in **"jwt_validation.py"**) altogether (it is probably better for future)?
@basilv @MCatherine1994

Reply:
Hi Ian, thanks for the explanation!!